theblues/plans.py

```diff
@@ -0,0 +1,50 @@
+from collections import namedtuple
+import datetime
+
+from theblues.errors import (
+    log,
+    ServerError,
+)
+from theblues.utils import (
+    ensure_trailing_slash,
+    make_request,
+)
+
+Plan = namedtuple('Plan', 'url plan created_on description price')
+
```

Comment:
Looks like the second element of the named tuple should be several elements? e.g.

```python
Plan = namedtuple('Plan', 'url',  'plan', 'created_on', 'description', 'price')
```

When you're adding below it looks like each of those is a separate element--am I misreading? Or am I misunderstanding named tuples?